README.md

```diff
@@ -40,20 +40,36 @@ print('3:', test.evaluate(lambda n: n**2), 'wrong')      # wrong
 test.save('dobro.test')
  ```
 
- - After the users executed their tests and submit the solution you could run the table.
+ - After the users executed their tests and submit the solution you could run the table (with sort options).
  ```python
  import os
  from jn_tester.professor.tables import view_complete_table
  
  # Keep in mind that you may need to pass the base path to look for all students files
  base_path = '/'.join(os.getcwd().split('/')[:-1]) + '/**/'
- view_complete_table('dobro.test', base_path=base_path)
+ 
+ sort = [('time', False), ('rank', True)]
+ view_complete_table('dobro', base_path=base_path, sort_by=sort)
+ ```
+ 
+ - After the visualization, there's option to export the results to .csv, .pkl and .html
```

Comment:
- After the visualization, it is possible to export the results to .csv, .pkl and .html